src/python/WMCore/WMSpec/Steps/Executors/CMSSW.py

```diff
@@ -53,12 +53,17 @@ def _setStatus(self, returnCode, returnMessage):
         """
         Set return code.
         """
-        self.setCondorChirpAttrDelayed('Chirp_WMCore_cmsRun_ExitCode', returnCode)
-        self.setCondorChirpAttrDelayed('Chirp_WMCore_%s_ExitCode' % self.stepName, returnCode)
-        if returnMessage and returnCode != 0:
-            self.setCondorChirpAttrDelayed('Chirp_WMCore_cmsRun_Exception_Message', returnMessage, compress=True)
-            self.setCondorChirpAttrDelayed('Chirp_WMCore_%s_Exception_Message' % self.stepName, returnMessage, compress=True)
+        # we will set Chirp_WMCore_cmsRun_ExitCode codes only if previous step was clean
+        if not self.failedPreviousStep:
+            logging.info("Step %s: Chirp_WMCore_cmsRun_ExitCod %s", self.stepName, returnCode)
```

Comment:
Typo. Should read `Chirp_WMCore_cmsRun_ExitCode`.
Similar to the line below.

Reply:
somehow my original push failed, now code does not have these logging statements at all.